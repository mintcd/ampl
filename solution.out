Artelys Knitro 14.0.0: 
=======================================
          Commercial License
         Artelys Knitro 14.0.0
=======================================

Knitro presolve eliminated 0 variables and 0 constraints.

concurrent_evals:        0
datacheck:               0
hessian_no_f:            1
The problem is identified as unconstrained.

Problem Characteristics                                 (   Presolved)
-----------------------
Objective goal:  Minimize
Objective type:  general
Number of variables:                                  2 (           2)
    bounded below only:                               0 (           0)
    bounded above only:                               0 (           0)
    bounded below and above:                          0 (           0)
    fixed:                                            0 (           0)
    free:                                             2 (           2)
Number of constraints:                                0 (           0)
    linear equalities:                                0 (           0)
    quadratic equalities:                             0 (           0)
    gen. nonlinear equalities:                        0 (           0)
    linear one-sided inequalities:                    0 (           0)
    quadratic one-sided inequalities:                 0 (           0)
    gen. nonlinear one-sided inequalities:            0 (           0)
    linear two-sided inequalities:                    0 (           0)
    quadratic two-sided inequalities:                 0 (           0)
    gen. nonlinear two-sided inequalities:            0 (           0)
Number of nonzeros in Jacobian:                       0 (           0)
Number of nonzeros in Hessian:                        3 (           3)

Knitro using the Interior-Point/Barrier Direct algorithm.

  Iter      Objective      FeasError   OptError    ||Step||    CGits 
--------  --------------  ----------  ----------  ----------  -------
       0    1.420313e+01   0.000e+00
       8    6.513981e-18   0.000e+00   5.716e-09   4.927e-05        0

EXIT: Locally optimal solution found.

Final Statistics
----------------
Final objective value               =   6.51398113864405e-18
Final feasibility error (abs / rel) =   0.00e+00 / 0.00e+00
Final optimality error  (abs / rel) =   5.72e-09 / 5.72e-09
# of iterations                     =          8 
# of CG iterations                  =          1 
# of function evaluations           =          9
# of gradient evaluations           =          9
# of Hessian evaluations            =          8
Total program time (secs)           =       0.013 (     0.000 CPU time)
Time spent in evaluations (secs)    =       0.000

===============================================================================

Knitro 14.0.0: Locally optimal or satisfactory solution.
objective 6.513981139e-18; feasibility error 0
8 iterations; 9 function evaluations

suffix feaserror OUT;
suffix opterror OUT;
suffix numfcevals OUT;
suffix numiters OUT;
