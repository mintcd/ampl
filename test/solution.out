SNOPT 7.5-1.2 : Optimal solution found.
1 iterations, objective 1.818989404e-12

Solution for a = 100:
Objective value: 0.000000
x: 100.000000
SNOPT 7.5-1.2 : Optimal solution found.
1 iterations, objective 0

Solution for a = 200:
Objective value: 0.000000
x: 200.000000
SNOPT 7.5-1.2 : Optimal solution found.
1 iterations, objective 0

Solution for a = 300:
Objective value: 0.000000
x: 300.000000
